README.md

```diff
@@ -42,27 +42,21 @@ Webpage: http://users.isy.liu.se/cvl/marda26/
 
 ### Using git clone
 
-1. Clone the GIT repository:
+1. Clone the GIT repository and submodules:
 
-   $ git clone https://github.com/martin-danelljan/ECO.git
+   $ git clone --recurse-submodules https://github.com/martin-danelljan/ECO.git
 
-2. Clone the submodules.  
-   In the repository directory, run the commands:
-
-   $ git submodule init  
-   $ git submodule update
-
-3. Start Matlab and navigate to the repository.  
-   Run the install script:
+1. Start Matlab/Octave and navigate to the repository path, run the install script:
```

Comment:
Just FYI. Markdown supports use same **1.** to create an [order list](https://guides.github.com/features/mastering-markdown/#)

I will still change to 1. 2. 3. as you wish.